pages/common/ia.md

```diff
@@ -0,0 +1,24 @@
+# ia
+
+> Command-line tool to interact with `archive.org`.
```

Comment:
I don't feel that `archive.org` needs to be surrounded by backticks anywhere on this page.

Reply:
I think it gives a nice touch to the page:
![image](https://user-images.githubusercontent.com/50295997/141676839-87f71b22-1d60-480a-abdd-6fe4aaf40548.png)

Reply:
@sbrl do you think we should change it anyways or can we merge?

Reply:
I'll merge because this PR was created 16 days ago.